src/pages/ConfidentialityPage/components/Confidentiality/ConfidentialityPart.scss

```diff
@@ -1,3 +1,7 @@
+:root {
+    --bg-color : #f8f8f8;
+    --orange-theme: #ffa600;
+}
```

Comment:
please change it to scss variables in the next pull request))
something similar to _variables.scss in yeti-react project)